mise.toml

```diff
@@ -0,0 +1,51 @@
+[tools]
+
+# Core dependencies
+go = "1.22.7"
+rust = "1.83.0"
+python = "3.13.0"
+uv = "0.5.5"
+jq = "1.7.1"
+shellcheck = "0.10.0"
+
+# Cargo dependencies
+"cargo:just" = "1.37.0"
+"cargo:svm-rs" = "0.5.8"
+
+# Go dependencies
+"go:github.com/ethereum/go-ethereum/cmd/abigen" = "1.10.25"
+"go:github.com/ethereum/go-ethereum/cmd/geth" = "1.14.7"
+"go:github.com/protolambda/eth2-testnet-genesis" = "0.10.0"
+"go:gotest.tools/gotestsum" = "1.12.0"
+"go:github.com/vektra/mockery/v2" = "2.46.0"
+"go:github.com/golangci/golangci-lint/cmd/golangci-lint" = "1.61.0"
+"go:github.com/mikefarah/yq/v4" = "4.44.3"
+
+# Python dependencies
+"pipx:slither-analyzer" = "0.10.2"
+"pipx:semgrep" = "1.90.0"
+
+# Foundry dependencies
+# Foundry is a special case because it supplies multiple binaries at the same
+# GitHub release, so we need to use the aliasing trick to get mise to not error
+forge = "nightly-7f41280ee071193557f73f16bae9aee9a5548ee8"
+cast = "nightly-7f41280ee071193557f73f16bae9aee9a5548ee8"
+anvil = "nightly-7f41280ee071193557f73f16bae9aee9a5548ee8"
+
+# Fake dependencies
+# Put things here if you need to track versions of tools or projects that can't
+# actually be managed by mise (yet). Make sure that anything you put in here is
+# also found inside of disabled_tools or mise will try to install it.
+asterisc = "1.1.1"
+kontrol = "1.0.53"
+binary_signer = "1.0.4"
+
+[alias]
+forge = "ubi:foundry-rs/foundry[exe=forge]"
+cast = "ubi:foundry-rs/foundry[exe=cast]"
+anvil = "ubi:foundry-rs/foundry[exe=anvil]"
+
+[settings]
+experimental = true
+pipx.uvx = true
+disable_tools = ["asterisc", "kontrol", "binary_signer"]
```

Comment:
How do these tools get installed now?

Reply:
`asterisc` isn't really a tool being installed, the version is just so that the `asterisc` CI job can verify that you haven't mucked with the file that's in the repo. `kontrol` I need to figure out separately, we didn't have an existing way to install `kontrol` because it relies on downloading `kup` first. It would be nice if kontrol had binary releases in GitHub or something. `binary_signer` also just downloads a python file from GitHub releases in CI.